src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -134,9 +134,8 @@ export const ArtworkDetailsAboutTheWorkFromPartnerFragmentContainer = createFrag
           city
         }
         profile {
-          __id
+          ...FollowProfileButton_profile
```

Comment:
This change allowed me to remove casting profile to `as any` above. But that is also duplicated some data here. I think it's still the correct way to go about definitions here, @damassi ?

Reply:
This is correct 👍